src/os/macosx/string_osx.cpp

```diff
@@ -191,8 +194,10 @@ static CTRunDelegateCallbacks _sprite_font_callback = {
 
 		/* Install a size callback for our special sprite glyphs. */
 		for (ssize_t c = last; c < i.first; c++) {
-			if (buff[c] >= SCC_SPRITE_START && buff[c] <= SCC_SPRITE_END) {
+			if (buff[c] >= SCC_SPRITE_START && buff[c] <= SCC_SPRITE_END && i.second->fc->MapCharToGlyph(buff[c], false) == 0) {
```

Comment:
When reading the PR, both description and code, I cannot figure out why only MacOS needs the fallback set to false in one instance. I am not actually sure what problem that actually solves, and why only for MacOS. No comment also explains that to me in a way I understand.

Mind adding some snippets somewhere to make that a bit more clear, or tell me: look, it is here, read (which is also fine :D ).

Reply:
The existing comment does already explain it, but not in a very obvious way. I can expand that.

I.e., we need to install the size callback for *sprite* glyphs, but not for font glyphs. And the font might or might not contain the special use chars.

Reply:
That still doesn't actually tell me why only MacOS needs this :) That is the part I am most confused about :)

Reply:
Because the other OS's just work differently?

Reply:
Haha, that is fair :) I can accept that as answer :)